pyvocz/views.py

```diff
@@ -39,6 +44,8 @@
 }
 
 
+cache = SimpleCache()
+
```

Comment:
Tady se trochu bojím. Podle [dokumentace](http://werkzeug.pocoo.org/docs/0.14/contrib/cache/#werkzeug.contrib.cache.SimpleCache) se SimpleCache v produkci moc nedoporučuje.
Co se stane, když přijde nějaký crawler a vyžádá si QR kódy pro všechna Pyva? Nevyčerpá to paměť?

Reply:
Pro představu:
* Kontejner s pyvo.cz má maximum celkem 256MB RAM a 2GB disku; v klidovém stavu aktuálně žere 85MB RAM.
* Můžu rozjet [Redis](https://docs.rosti.cz/storages/redis/) nebo [Memcached](https://docs.rosti.cz/storages/memcached/), je-li třeba (ale mělo by to umět fungovat i bez toho, ať se dá přispívat).

Nevím jak se chová SimpleCache, ale než to mergnu tak je to potřeba trochu promyslet.

Reply:
Už jsem se na to díval předtím, ale veškerá omezení a základní nastavení mi přišla pro daný účel bezpečná. Při zaplnění cache a průměrné velikosti qrcode PNG něco přes 500 bajtů by to mělo zabrat něco přes 250 kB (max.) 

Více zde: http://werkzeug.pocoo.org/docs/0.14/contrib/cache/#werkzeug.contrib.cache.SimpleCache

Reply:
OK, jestli ses na to díval, není problém :)

Reply:
Jsem tam dal radši i ty defaulty + upozornění...